app/models/prison.rb

```diff
@@ -79,9 +79,9 @@ def summary
       summary = unfiltered_offenders.group_by do |offender|
         if offender.inside_omic_policy?
           allocatable = if womens?
-                          offender.has_case_information? && offender.complexity_level.present?
+                          offender.probation_record.present? && offender.complexity_level.present?
                         else
-                          offender.has_case_information?
+                          offender.probation_record.present?
```

Comment:
At one point I think we talked about the Offender object itself _telling us_ what it is...

e.g. `offender.allocatable?`

Rather than making it the responsibility of the Prison model to decide when an Offender is in an 'allocatable' state.